### PR TITLE
Deduplicate websphere-to-eap6or7

### PR DESCRIPTION
**Pull request by OndraZizka**

### CODE REVIEW

File: rules-reviewed/README.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+This directory contains default set of Windup rules distributed with RHAMT.
```
Comment (jsight) [inline rules-reviewed/README.txt]: Good idea... although maybe once this current round of fixes is done, we can fix the name. rules-reviewed shouldn't stay here forever, IMO.

**Comment by jsight** [comment]
-1 on deduplication in this way. This duplicates a lot of links and isn't really necessary.

I think the duplication in the slide should only focus on the duplication between classification and hint.

File: rules-reviewed/README.txt (hunk 1 above)
Comment (OndraZizka) [inline rules-reviewed/README.txt]: haha

**Comment by OndraZizka** [comment]
Are you sure? I can barely read the screenshot but I see one entry for EAP 7 and one for EAP 6.

**Comment by jsight** [comment]
@OndraZizka - I think we need a separate PR to insure that the correct target is run (eap6 or 7 vs just "eap").

**Pull request merged**